guides/it/linux_av.md

```diff
@@ -194,41 +216,8 @@ ClamScanQueue: stopped
 Clamonacc: stopped
 ```
 
-That likely indicates you need to increase the number of inotify watchers available as mentioned above.
-
-We can test this is running correctly by downloading a test file:
-
-```
-$ wget www.eicar.org/download/eicar.com
-$ ls | grep eicar
-$ sudo ls /root/quarantine
-eicar.com
-```
+You may need to increase the number of inotify watchers available from the default 65536.
```

Comment:
You WILL need to increase the watch count ; just installing one version of NodeJS in your home folder will eat 36,000 inodes without breaking a sweat.

Bear in mind that each watch eats a kilobyte of RAM.

Reply:
This whole setup feels a little iffy/brittle still...

Reply:
I can add the sysfs config to the Ansible playbook ... but I have concerns about the resource consumption.

My old workstation has 4.2M files in `$HOME` - if you can't watch them all, `clamonacc` doesn't start. So let's say 5M watches to be safe and give some room for VSCode etc to watch files too. On the issue ThinkPad, that's a third of the RAM.... and then there's the matter of it eating a whole CPU core (and presumably winding that back to 30% just means the scanning queue is being partially nelected?)

Maybe a sensible happy medium like "Watch `~/Downloads` and `/tmp' and also do a periodic full scan of `/home`" would be better ...

Reply:
If the answer is we don't need to run ClamAv that would be smashing...